docs/sql-ref-syntax-ddl-create-table.md

```diff
@@ -19,4 +19,9 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`CREATE TABLE` statement is used to create a table in an exsisting database. 
+
+The INSERT statements:
+* [CREATE TABLE USING DATASOURCE](sql-ref-syntax-ddl-create-table-datasource.html)
+* [CREATE TABLE USING HIVE FORMAT](sql-ref-syntax-ddl-create-table-hiveformat.html)
```

Comment:
We need to add `CREATE TABLE LIKE`, too? https://github.com/apache/spark/blob/18e8d1d5b2bab3194cb2118d3da16109922a578d/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4#L129